### PR TITLE
ascanrulesBeta: Check more hidden files

### PR DESCRIPTION
**Pull request by security-companion**
I compared the current hidden_files.json with the current snallygaster repo and found some checks still missing in the json. This PR adds the missing ones.

### CODE REVIEW

**Comment by kingthorin** [comment]
To address the DCO requirement you'll need to sign-off the commit(s):

- https://github.com/zaproxy/zaproxy/blob/develop/CONTRIBUTING.md#developer-certificate-of-origin
- https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---signoff

**Comment by kingthorin** [comment]
There are separate scan rules for elmah and .env 😉

File: addOns/ascanrulesBeta/src/main/zapHomeFiles/json/hidden_files.json [hunk 1]
```diff
@@ -200,12 +200,37 @@
...
+      "source":"snallygaster"
+    },
     {
       "path":"adminer.php",
       "content":["adminer.org"],
       "type":"adminer",
       "source":"snallygaster"
     },
+    {
+      "path":"elmah.axd",
```
Comment (thc202) [inline addOns/ascanrulesBeta/src/main/zapHomeFiles/json/hidden_files.json:222]: There's already a specific scan rule checking this file (same for env).

**Comment by security-companion** [comment]
@kingthorin Well then never mind, didn't know about this.
Thanks anyway.

**Comment by kingthorin** [comment]
Still worth doing the php and vBulletin entries.

**Comment by security-companion** [comment]
Okay thanks,
I wanted to check https://github.com/zaproxy/zaproxy/blob/develop/CONTRIBUTING.md#developer-certificate-of-origin but it only leads to a 404 error. Am I doing something wrong?

**Comment by thc202** [comment]
It's using other branch, check in main: https://github.com/zaproxy/zaproxy/blob/main/CONTRIBUTING.md#developer-certificate-of-origin

**Comment by kingthorin** [comment]
Looks like I need to updated my saved reply, thanks for the heads up 👍

File: addOns/ascanrulesBeta/src/main/zapHomeFiles/json/hidden_files.json [hunk 2]
```diff
@@ -243,6 +249,34 @@
...
+      "type":"phpinfo",
+      "source":"snallygaster"
+    },
+    {
+      "path":"test.php",
+      "content":["phpinfo()"],
+	  "links":["https://www.php.net/manual/en/function.phpinfo.php"],
+      "type":"phpinfo",
+      "source":"snallygaster"
+    },
```
Comment (kingthorin) [inline addOns/ascanrulesBeta/src/main/zapHomeFiles/json/hidden_files.json:280]:
```suggestion
    }
```

**Comment by kingthorin** [comment]
@security-companion could you re-add the CHANGELOG entry? Otherwise I think this is ready.

**Comment by security-companion** [comment]
@kingthorin sorry, the changelog got lost on the way. I readded it now.

**Comment by security-companion** [comment]
Now I forgot to sign the commit, I guess I need to fix this?

**Review by kingthorin — APPROVED** [review]
Thanks

**Comment by kingthorin** [comment]
Yes. But if you’d rather I do it and squash them all too.

File: addOns/ascanrulesBeta/CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ## Unreleased
 ### Changed
 - Maintenance changes.
+- Adding more checks to hidden-files add-on
```
Comment (thc202) [inline addOns/ascanrulesBeta/CHANGELOG.md:9]: `hidden-files add-on` → `Hidden File Finder scan rule.`

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Comment by security-companion** [comment]
@thc202 Thanks to you for merging.

**Pull request merged**